keyboards/handwired/nozbe_macro/rules.mk

```
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+# Build Options
+#   comment out to disable the options.
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```